cmd/snap/cmd_buy_test.go

```diff
@@ -350,29 +375,31 @@ Once completed, return here and run 'snap buy hello' again.`)
 	c.Check(s.Stderr(), check.Equals, "")
 }
 
-func (s *BuySnapSuite) TestBuyCancel(c *check.C) {
-	mockServer := &buyTestMockSnapServer{
-		ExpectedMethods: expectedMethods{
-			"GET": &expectedMethod{
-				"/v2/find":      buySnapFindURL(c),
-				"/v2/buy/ready": buyReady(c),
-			},
-		},
-		Checker: c,
-	}
-	defer mockServer.checkCounts()
-	s.RedirectClientToTestServer(mockServer.serveHttp)
-
-	// Decline the payment
-	fmt.Fprint(s.stdin, "no\n")
-
-	rest, err := snap.Parser().ParseArgs([]string{"buy", "hello"})
-	c.Assert(err, check.NotNil)
-	c.Check(err.Error(), check.Equals, "aborting")
-	c.Check(rest, check.DeepEquals, []string{"hello"})
-	c.Check(s.Stdout(), check.Equals, `Do you want to buy "hello" from "canonical" for 2.99GBP? (Y/n): `)
-	c.Check(s.Stderr(), check.Equals, "")
-}
+//func (s *BuySnapSuite) TestBuyCancel(c *check.C) {
```

Comment:
Actually this test no-longer makes any sense. There's no special handling for SIGINT/TERM - snap just exits, so there's nothing to test.